target/scripts/startup/setup.d/dmarc_dkim_spf.sh

```diff
@@ -97,8 +97,14 @@ function _setup_policyd_spf
 policyd-spf    unix  -       n       n       -       0       spawn
     user=policyd-spf argv=/usr/bin/policyd-spf
 EOF
+
+  sedfile -i -E \
+    's|^(smtpd_recipient_restrictions.*reject_unauth_destination)(.*)|\1, check_policy_service unix:private/policyd-spf\2|' \
+    /etc/postfix/main.cf
+  # SPF policy settings
+  postconf 'policyd-spf_time_limit = 3600'
+EOF
```

Comment:
EOF was advised to be dropped for this suggestion.

Reply:
I should have read this more carefully, sorry 😨

Fixed in f62855f89dc61e272dd87791f518b8813a6f2b35 in PR #3261